lib/src/ios/backpack-react-native/backpack-react-native.podspec

```diff
@@ -1,19 +1,19 @@
 require 'json'
 
-package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+package = JSON.parse(File.read(File.join(__dir__, '../../../../package.json')))
 
 Pod::Spec.new do |s|
-  s.name         = "react-native-bpk-component-calendar"
+  s.name         = "backpack-react-native"
   s.version      = package['version']
-  s.summary      = "Backpack calendar component for React Native"
+  s.summary      = "Backpack for React Native"
 
   s.authors      = { "backpack" => "backpacksquad@skyscanner.net" }
   s.homepage     = "https://backpack.github.io/components/calendar?platform=ios"
   s.license      = "MIT"
   s.platform     = :ios, "11.0"
 
   s.source       = { :git => "https://github.com/Skyscanner/backpack-react-native.git" }
-  s.source_files  = "src/ios/CalendarBridge/**/*.{h,m}"
+  s.source_files  = "**/*.{h,m}"
 
   s.dependency 'React'
   s.dependency 'Backpack', '~> 26.0'
```

Comment:
Better make this `'~> 26.1.0'` since that version is the latest

Reply:
Good point, I'll update when I finish this as it might stay here for a while.